pkg/maps/maps.go

```diff
@@ -39,3 +39,14 @@ func Values[M ~map[K]V, K comparable, V any](m M) []V {
 func Keys[M ~map[K]V, K comparable, V any](m M) []K {
 	return maps.Keys(m)
 }
+
+// Merge create a new map merge all key/value pairs from base and override.
+// When a key in override is already present in base,
+// the value in base will be overwritten by the value associated
+// with the key in override.
+func Merge[M1 ~map[K]V, M2 ~map[K]V, K comparable, V any](base M1, override M2) M1 {
```

Comment:
```suggestion
func Merged[M1 ~map[K]V, M2 ~map[K]V, K comparable, V any](base M1, override M2) M1 {
```

Makes it more clear its a copy. Or `MergeCopy`

Reply:
thanks